ci/playbooks/kuttl/trigger-kuttl-tests.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: 'Install {{ operator_name }} operator'
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_{{ operator_name }}.yml'
+
+- name: Run operator kuttl tests
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_{{ operator_name }}_kuttl.yml'
```

Comment:
I'm not too familiar with the ci-framework at this point, but I suppose that including tasks from `make_{{ operator_name }}_kuttl.yml` is equivalent to calling `make {{ operator_name}}_kuttl` from the install_yamls makefile, right?

Reply:
Yes correct. these tasks files are generated by [install_yamls](https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/install_yamls) role in the run time into install_yamls_makes role.

We use the generated role to call these tasks. 
For example: https://logserver.rdoproject.org/63/163/9a4559d5176e9bac34f183332dc156d18356ef5a/github-check/cifmw-kuttl/84305ff/ci-framework/artifacts/roles/install_yamls_makes/tasks/

and make_cinder_kuttl.yaml example: https://logserver.rdoproject.org/63/163/9a4559d5176e9bac34f183332dc156d18356ef5a/github-check/cifmw-kuttl/84305ff/ci-framework/artifacts/roles/install_yamls_makes/tasks/make_cinder_kuttl.yml